warp/yul/AstTools.py

```diff
@@ -413,3 +295,116 @@ def get_word(self, start: int) -> str:
     def get_name(self, name):
         name = "_".join(re.findall("[A-Z][^A-Z]*", name))
         return name.lower()
+
+
+class YulPrinter(AstVisitor):
+    def format(self, node: ast.Node, tabs: int = 0) -> str:
+        return self.visit(node, tabs)
+
+    def visit_typed_name(self, node: ast.TypedName, tabs: int = 0) -> str:
+        return f"{node.name} : {node.type}"
```

Comment:
Just a suggestion, since Uint256 is the default type in the EVM dialect of Yul that we use, we can simply omit it. It leads to a bit less visual noise.

Reply:
Also true, I'll change it as well